actions/lib/action.py

```diff
@@ -70,8 +70,10 @@ def cf_connect(self):
         return boto.cloudformation.connect_to_region(region, **self.credentials)
 
     def r53_connect(self):
-        del self.credentials['region']
-        return boto.route53.connection.Route53Connection(**self.credentials)
+        route53_credentials = {
+            key: self.credentials[key] for key in self.credentials if key != 'region'
```

Comment:
IIRC, dict comprehensions are supported in Python >= 2.7 so we should probably be fine here (just thinking out loud).

Reply:
Another option would also be to explicitly assign `self.route53_credentials` inside a constructor where we assign `self.credentials` and use that there.